editor/src/components/canvas/canvas-strategies/strategies/flow-reorder-strategy.spec.browser2.tsx

```diff
@@ -596,6 +597,8 @@ describe('Flow Reorder Strategy (Mixed Display Type)', () => {
   })
 
   describe('with fragments as siblings', () => {
+    setFeatureForBrowserTests('Fragment support', true)
```

Comment:
I turned on fragment support in all of these tests I touched. _maybe_ it regresses the feature switch off state, but I think this PR was the last blocker before we can kill the Fragment feature switch altogether! cc @gbalint